14/umbraco-cms/fundamentals/setup/server-setup/azure-web-apps.md

```diff
@@ -54,6 +54,8 @@ You can also copy the following JSON directly into your Azure Web App configurat
 }
 ```
 
+Also remember to add environment variable `ASPNETCORE_ENVIRONMENT` witch value `Development`, `Staging` og `Production`.
```

Comment:
A suggestion to rewrite a bit to improve readability - let me know what you think.

```suggestion
Remember to add an `ASPNETCORE_ENVIRONMENT` variable with values `Development`, `Staging`, or `Production`.
```